### ISSUE TITLE
Screenshot or video of kinect functions 

### ISSUE THREAD

**Issue by joepal1976** [makehumancommunity/makehuman-plugin-for-blender#35]
JCPalmer: Any chance you could provide a screenshot or a video of how it looks when the kinect integration is running? I'm writing release notes for the upcoming MH release (which will bundle this plugin), and would like to have screenshots of new functions.

**Comment by joepal1976** [comment]
Work in progress release notes are here: http://www.makehumancommunity.org/wiki/Releases:120a1

**Comment by Palmer-JC** [comment]
Will get a compact video of a trimmed, anti-jitterred, action assigned by say, Saturday.  Maybe 20 seconds.

FYI, nothing really happens on screen while recording is in progress.  The reasons are many:
- A kinect can record up to 6 bodies.  After recording, you can then assign each part to each MH (even across .blend file, if you hit the refresh button).  Tough to set all that up prior to recording.
- It simplified the code, and actually helped figure things out during dev.  I could run assignment multiple times, & with the help of temp UI, change how the data was processed.
-  It did not see important to have on-screen, unless the monitor was pointed the same way as the sensor.  I do not have that here.

Also, what I plan to do is make a video from a webgl scene, where the MH character gives a talking instructional video on using.  This would allow me to test out my entire pile line, and screw around with stuff like having to duck out of the way of signs with bullet points on them.

**Comment by joepal1976** [comment]
Yeah, we should refactor the code into a more sensible code file division. I'm thinking at least panels should be in separate files, possibly also operators. 

Also, the current directory hierarchy doesn't make much sense. It was made when all the plugin did was to sync the shape of the mesh. Since a lot more is happening now, we should probably have a more relevant directory hierarcy.

**Comment by Palmer-JC** [comment]
Maybe just create a new repo.  decorate / split out as now needed.  Everything on Master branch.  I will continue to play with my local repo.  When done, I will clone directly, and replace my changed files.

Just nuke old repo, when version.
